src/Makefile

```diff
@@ -0,0 +1,13 @@
+build: rolctl rol
+
+rolctl: webapi/swagger/swagger.json
+	swagger generate cli -f webapi/swagger/swagger.json -A rol-cmd #--skip-validation
```

Comment:
I think we can update the readme with information that we need the swagger utility, and indicate how to get this utility, link or instruction.